### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by prekpara** [Fugiman/google-meet-grid-view#344]
The extension stopped working. It no longer puts people in ABC order and when using it puts everyone in 1 long vertical line and not in a grid view. 

**Screenshots**

**Software used:**
 - OS [e.g. Windows 10]:
 - Browser [e.g. Chrome, Firefox]:
 - Version [e.g. 22]:
 - Extension [e.g. Official Chrome Extension, Tampermonkey]:
 - Grid View Version [e.g. 1.19, 1.23]:

**Additional context**

**Comment by Jkarhani** [comment]
Yes, it seems that google stopped it by updating itself..we should ask google to enable it back.. Please let us all complain.

**Comment by Jkarhani** [comment]
Windows 10
Google chrome 
(88.0.4) 64 bits
Extension google chrome official
Grid view version (1.37) - Same for the grid view fix (1.41)

**Comment by Eagle-Man** [comment]
From the Grid View (Fix) GitHub page:

> UPDATE 02/11/2021:
> It turns out that Google is testing an update of Meet that disables the Grid View extension. This is why sometimes it works and sometimes not (you see the boxes in a vertical bar). If you seriously need Grid View right now, you might consider refreshing a few times the browser tab (you should end up loading the old compatible version at a certain point).
> I am really sorry but I have to temporarily remove the extension from Chrome Web Store. In fact, updating Grid View would require a huge amount of work or may not be feasible at all. Please, remember that Google Meet introduced native grid view (up to 49 tiles).

**Comment by Eagle-Man** [comment]
@Jkarhani The solution is NOT to ask Google to "put it back." There's no way they're going to customize their code for one third-party extension. The real solution is to tell Google what would make Meet better for you; what features you want to see in Meet. And if enough people do this, maybe we'll see some of them native to Meet where they can't be broken again.

**Comment by Jkarhani** [comment]
@Eagle-Man thank you, yes I do realize this. Yes , we sent a report to MR Google & telling all the teachers to do so.
